cmd/fs-v1-metadata.go

```diff
@@ -286,16 +286,16 @@ func newFSFormatV2() (format *formatConfigV1) {
 }
 
 // Checks if input format is version 1 and 2.
-func isFSValidFormat(formatCfg *formatConfigV1) bool {
-	// Supported format versions.
-	var supportedFormatVersions = []string{
+func isKnownFSFormat(formatCfg *formatConfigV1) bool {
+	// Known format versions.
+	var knownFormatVersions = []string{
```

Comment:
You could move this as global of the function for managing the array better for future versions.